### ISSUE TITLE
Links in sidebar don't match text (are off by one row)

### ISSUE THREAD

**Issue by adrian5** [webpack/webpack.js.org#3322]
If you go into the documentation on loaders, e.g. [style-loader](https://webpack.js.org/loaders/style-loader/), all navigation links from _raw-loader_ downwards don't match their label. I think they're misaligned by one row.

**Comment by montogeek** [comment]
@EugeneHlushko Seems like the same problem :D

**Comment by godcrampy** [comment]
I tried to recreate the problem, and here's what I found. If you visit the page using a link, like by clicking the above link or actually pasting the URL in the browser you'll find the links are one off. On the other hand if you navigate to the page by visiting the loaders main [page](https://webpack.js.org/loaders/), and then clicking in the style-loader link in the sidebar, the links are fine.

On running the site locally, it turns out that both of the above mentioned methods return properly mapped links. _Interesting_...
